src/components/top-matter/MapSplitMatter.vue

```diff
@@ -245,6 +259,34 @@ export default defineComponent({
       this.animateMarkers();
     },
 
+    async fetchPoints() {
+      const zoom = "18";
+      let minLat = -90;
+      let  maxLat = 90;
+      let minLon = -180;
+      let maxLon = 180;
+
+      // Extend bounds by 25% beyond the map
+      const latDiff = Math.abs(maxLat - minLat);
+      const lonDiff = Math.abs(maxLon - minLon);
+      minLat -= latDiff * 0.25;
+      maxLat += latDiff * 0.25;
+      minLon -= lonDiff * 0.25;
+      maxLon += lonDiff * 0.25;
+
+      // Get bounds with expanded margins
+      const bounds = this.boundsToStr({ minLat, maxLat, minLon, maxLon });
+
+      // Make API call
+      const url = API.Q(API.MAP_CLUSTERS(), { bounds, zoom });
+
+      // Params have changed, quit
+      const res = await axios.get(url);
+      this.points = res.data.map((c) => {
+        return [c.center[0], c.center[1], 1]
+      });
```

Comment:
If we are loading this current screen only, this might as well be a computed prop derived from `this.clusters`

Reply:
As said in the long response. From my experience the heatmap is capable to handle much more Datapoints than the markers. 

If this observation holds I think it makes sense to seperate this two data sources.